### DISCUSSION TITLE
Persisted plugin

### DISCUSSION THREAD

**Discussion by semi710** [glacambre/firenvim#1544] [Q&A] [answered]
https://github.com/olimorris/persisted.nvim/discussions/77#discussioncomment-6298868

does firenvim have a specific filetype or any difference from other file type ?

**Comment by glacambre** [comment, marked as answer]
No, the filetype can change depending on the website you are editing. All of the files are created in $XDG_RUNTIME_DIR/firenvim , so if persisted.nvim allows ignoring files depending on their paths, you could rely on that.
